requirements.py2.txt

```diff
@@ -0,0 +1 @@
+configparser==3.5.0
```

Comment:
I know six provides a link to the configparser, but using this I can keep the code more clean (one day Apache Beam will support Python 3, one day...).